chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/listing/pending/PendingCertifiedProductDTO.java

```diff
@@ -611,23 +603,30 @@ public PendingCertifiedProductDTO(PendingCertifiedProductEntity entity) {
 
         this.uploadDate = entity.getCreationDate();
 
-        Set<PendingCertifiedProductTestingLabMapEntity> testingLabs = entity.getTestingLabs();
-        if (testingLabs != null && testingLabs.size() > 0) {
-            for (PendingCertifiedProductTestingLabMapEntity testingLab : testingLabs) {
+        Set<PendingCertifiedProductTestingLabMapEntity> testingLabEntities = entity.getTestingLabs();
+        if (testingLabEntities != null && testingLabEntities.size() > 0) {
+            for (PendingCertifiedProductTestingLabMapEntity testingLab : testingLabEntities) {
                 this.testingLabs.add(new PendingCertifiedProductTestingLabDTO(testingLab));
             }
         }
 
-        Set<PendingCertifiedProductQmsStandardEntity> qmsStandards = entity.getQmsStandards();
-        if (qmsStandards != null && qmsStandards.size() > 0) {
-            for (PendingCertifiedProductQmsStandardEntity qmsStandard : qmsStandards) {
+        Set<PendingCertifiedProductQmsStandardEntity> qmsStandardEntities = entity.getQmsStandards();
+        if (qmsStandardEntities != null && qmsStandardEntities.size() > 0) {
+            for (PendingCertifiedProductQmsStandardEntity qmsStandard : qmsStandardEntities) {
                 this.qmsStandards.add(new PendingCertifiedProductQmsStandardDTO(qmsStandard));
             }
         }
 
-        Set<PendingCertifiedProductTargetedUserEntity> targetedUsers = entity.getTargetedUsers();
-        if (targetedUsers != null && targetedUsers.size() > 0) {
-            for (PendingCertifiedProductTargetedUserEntity tu : targetedUsers) {
+        Set<PendingListingMeasureEntity> measureEntities = entity.getMeasures();
+        if (measureEntities != null && measureEntities.size() > 0) {
+            for (PendingListingMeasureEntity measureEntity : measureEntities) {
+                this.measures.add(new PendingCertifiedProductMeasureDTO(measureEntity));
+            }
+        }
+
+        Set<PendingCertifiedProductTargetedUserEntity> targetedUserEntities = entity.getTargetedUsers();
+        if (targetedUserEntities != null && targetedUserEntities.size() > 0) {
+            for (PendingCertifiedProductTargetedUserEntity tu : targetedUserEntities) {
                 this.targetedUsers.add(new PendingCertifiedProductTargetedUserDTO(tu));
             }
```

Comment:
Might want to consider using `map` to do this.

Reply:
I agree..  I guess I just wanted the new section for measures to be the same as all the old sections for all the other things. And I didn't really want to change all those old sections because I hope with the "rewrite the upload" work that all of the pending* classes will eventually disappear. So I didn't do anything for this comment but I can/will if you would prefer.